### PR TITLE
Feature/implement task 1

### PR DESCRIPTION
**Pull request by roman-oleynik**
Roman Oleynik

### CODE REVIEW

**Comment by Starotitorov** [comment]
Remove submodule teach-me-skills-javascript-basics from pull request changes. Look at changes https://github.com/nondont/teach-me-skills-javascript-basics/pull/2/files

File: RomanOleynik/css/style.css [hunk 1]
```diff
@@ -0,0 +1,115 @@
+
+html, body, div, span,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+img, ins, kbd, q, s, samp,
+small, strike, strong,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td {
+margin: 0;
```
Comment (Starotitorov) [inline RomanOleynik/css/style.css]: Please, format css. It is really hard to read this.

**Comment by nondont** [comment]
Remove submodule teach-me-skills-javascript-basics from pull request changes. Look at changes https://github.com/nondont/teach-me-skills-javascript-basics/pull/2/files

**Comment by nondont** [comment]
Read about BEM naming conventions and fix class names according to them. We discussed it on previous lesson.

**Pull request closed without merging**